source/common/http/conn_pool_grid.cc

```diff
@@ -167,12 +165,11 @@ void ConnectivityGrid::WrapperCallbacks::signalFailureAndDeleteSelf(
   if (callbacks != nullptr) {
     ENVOY_LOG(trace, "Passing pool failure up to caller.");
     std::string failure_str;
-    if (prev_tcp_pool_failure_reason_.has_value()) {
+    if (prev_pool_failure_reason_.has_value()) {
```

Comment:
the comment below doesn't stand with current changes, and it could be TCP / QUIC

Reply:
fixed, thanks for catching it!